migrator/init.lua

```diff
@@ -293,24 +320,7 @@ end
 local function init(opts)
     -- Create space for storing applied migrations.
     if opts.is_master then
-        box.schema.sequence.create('_migrations_id_seq', { if_not_exists = true })
-        box.schema.create_space('_migrations', {
-            format = {
-                {'id', type='unsigned', is_nullable=false},
-                {'name', type='string', is_nullable=false},
-            },
-            if_not_exists = true,
-        })
-        box.space._migrations:create_index('primary', {
-            sequence = '_migrations_id_seq',
-            if_not_exists = true,
-        })
-        -- Workaround for https://github.com/tarantool/ddl/issues/122
-        -- If index is created by ddl, sequence is not set. Check and update is required.
-        if box.space._migrations.index.primary ~= nil
-        and box.space._migrations.index.primary.sequence_id == nil then
-            box.space._migrations.index.primary:alter({sequence = '_migrations_id_seq'})
-        end
+        create_space_for_storing_applied_migrations()
```

Comment:
And do we need this one after update?

Reply:
Maybe it's simpler to just move to `apply` since this one triggers on master switch.

Reply:
Fixed